rpc/openapi/openapi.yaml

```
@@ -1002,6 +1002,50 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  /prove_shares:
+    get:
+      summary: Prove shares for a given share range
+      description: |
+        Generates a shares to data root inclusion proof
+        for a set of shares referenced by their range and
+        the block height they belong to.
```

This reads poorly. Perhaps it can be rephrased:

```suggestion
        Generates a proof of inclusion for a range of shares to the data root.
        Note: shares are referenced by their range: startShare to endShare.
        The share range is end exclusive.
```